### PR TITLE
Remove yarn-lock and upgrade node to ^8 and npm to ^5.

### PR DESCRIPTION
**Pull request by jelliotartz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
     "stage": "NODE_ENV=staging npm run build && publisssh ./dist zooniverse-static/preview.zooniverse.org/pfe-lab"
   },
   "engines": {
-    "node": "~7.10.0",
-    "npm": "~4.2.0"
+    "node": "^8.0.0",
```
Comment (srallen) [inline package.json]: The latest node and npm are 8.1.2 and 5.0.3 respectively, so we might as well bump to that. Let's also use tilde prefixes to stay consistent with PFE.

**Comment by mcbouslog** [comment]
Closes #39

**Comment by mcbouslog** [comment]
strange it has conflict with a file we're deleting, but could you try rebasing when you get a chance?

**Comment by mcbouslog** [comment]
this LGTM. @srallen I'll merge later today or feel free to, just mentioning as you have pending requested changes, didn't want to "press the button" without a note.

**Pull request merged**